src/AppBundle/Command/AddUserCommand.php

```diff
@@ -66,19 +73,6 @@ protected function configure()
     }
 
     /**
-     * This method is executed before the interact() and the execute() methods.
-     * It's main purpose is to initialize the variables used in the rest of the
-     * command methods.
-     *
-     * Beware that the input options and arguments are validated after executing
-     * the interact() method, so you can't blindly trust their values in this method.
```

Comment:
you should copy this comment in the DeleteUserCommand then, otherwise we don't have the complete explanation about ``interact`` anymore